api/nakadi-event-bus-api.yaml

```diff
@@ -29,11 +29,12 @@ info:
     before persisting it. Usually enrichment affects the metadata of an Event but is not limited to.
```

Comment:
Generally:  The API now looks much more consistent and has profited by uniform usage of EventType and removing the Topic resource aspects. 
Pls. find some new comments below. 
Pls. also take care that all other agreed comments of the former review will go into the next API version.